lib/plugins/create/create.js

```diff
@@ -16,6 +16,7 @@ const validTemplates = [
   'aws-groovy-gradle',
   'aws-java-maven',
   'aws-java-gradle',
+  'aws-kotlin-jvm-maven',
```

Comment:
The current naming convention is `provider-language/runtime-buildenv`.

Maybe we should rename the template to `aws-kotlin-maven` (since AFAIK AWS uses the `jvm` behind the scenes)?

Reply:
I've added JVM part because Kotlin can compile to Java (JVM) or JS (Node.js). In future plans, I wanted to add also Kotlin -> JS, hence the name.